apps/condo/domains/notification/schema/SyncDeviceService.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generated by `createservice notification.SyncDeviceService --type mutations`
+ */
+
+const get = require('lodash/get')
+
+const { GQLCustomSchema, getById } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/notification/access/SyncDeviceService')
+const { Device: DeviceAPI } = require('@condo/domains/notification/utils/serverSchema')
+
+const { PUSH_TRANSPORT_TYPES } = require('../constants/constants')
+
+const resolveSyncDevice = async (parent, args, context) => {
+    const { data: { dv, sender, deviceId, pushToken, pushTransport, meta } } = args
+    const userId = get(context, 'authedItem.id', null)
+    const owner = userId ? { disconnectAll: true, connect: { id: userId } } : null
+    const attrs = { dv, sender, deviceId, pushToken, pushTransport, meta, owner }
+    const where = { deviceId, pushTransport }
+    const data = await DeviceAPI.updateOrCreate(context, where, attrs)
+    const result = await getById('Device', data.id)
+
+    return result
+}
```

Comment:
Extracted resolver is inconsistent with style of other resolvers
Does it feels ok to others?

Reply:
fixed